### PR TITLE
WEB-5600 managed to replace http with request

### PR DESCRIPTION
**Pull request by t1gr0u**
#### What are the links to relevant tickets?

https://hxshortbreaks.atlassian.net/browse/WEB-5600
#### What does this PR do?

Replaces the _HTTP_ node module with _request_
#### What functional/unit tests does this PR have?

N/A
#### How should a developer review this?

Run the test.
#### Any background context you want to provide?

It seams that the HTTP node module is keeping the connection open.
#### Screenshots (if appropriate)

N/A

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [ ] I've updated documentation with any changes to procedures.
- [ ] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.
#### Reviewer 1 @panicnot
- [x] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!
#### Reviewer 2 @notABluesSinger
- [x] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [ ] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by notABluesSinger** [comment]
The reason we had the `request.end()` and non closed connection was to try and implement _fire and forget_ functionality.

With this change what happens if loggly is on a go real go slow? Not failed connections, just taking an age to respond (we had this issue on HXML a few weeks back and the way revolver works it did not get effected)
How does it effect async performance?

File: test/logglyInterfaceTest.js [hunk 1]
```diff
@@ -17,41 +17,36 @@ function loadTestResource( resource ) {
...
-		beforeEach( function( done ) {
-			httpsStub = sinon.stub( https, 'request', function() {
-				return {
-					write: function() {},
-					end: function() {}
-				}
-			} );
-			done();
-		} );
+	describe( '#log', function() {
```
Comment (notABluesSinger) [inline test/logglyInterfaceTest.js]: I prefer to use `context` over nested `describe`s, just cleaner overall

**Comment by notABluesSinger** [comment]
Could you add some more notes on the jira as to why the plan change.

File: test/logglyInterfaceTest.js (hunk 1 above)
Comment (t1gr0u) [inline test/logglyInterfaceTest.js]: Done

**Comment by t1gr0u** [comment]
Will do, once I've got the Revolver Pull request going.

**Pull request merged**